js_modules/dagster-ui/packages/ui-core/src/asset-graph/sidebar/AssetSidebarNode.tsx

```diff
@@ -124,7 +127,15 @@ const AssetSidebarAssetLabel = ({
       <FocusableLabelContainer
         isSelected={isSelected}
         isLastSelected={isLastSelected}
-        icon={<StatusDot node={node} />}
+        icon={
+          featureEnabled(FeatureFlag.flagUseNewObserveUIs) ? (
+            <div style={{marginLeft: -8, marginRight: -8}}>
```

Comment:
This is pretty hacky.... but not unique 😬 ... 
<img width="461" alt="Screenshot 2025-05-13 at 1 37 58 PM" src="https://github.com/user-attachments/assets/0db7325c-8eb2-4df2-a698-0a112fbffa18" />